ImP/preProcessing/testImageSimilarity.py

```diff
@@ -0,0 +1,27 @@
+import unittest 
+from preProcessor import PreProcessor 
```

Comment:
Since this may normally be run as following:
```
python3 -m ImP.preProcessing.testImageSimilarity
```
I would change this to a relative import:
```
from .preProcessor import PreProcessor
```